### DISCUSSION TITLE
No Power to display

### DISCUSSION THREAD

**Discussion by MattSpeke** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#583] [General]
Hi, I have built 2 veresions of the V2 PCB. Using headers H1A/B and H2A/B. Both versions are using the red LLC. I can access the web page both via USB and when connected to the pump. However I have no display on the pump. No red  light to say it has power and no control of the pump. I have the 560ohm resistors and capacitors on the PCB as I ordered it pre built with them fitted.

![20230805_171352671_iOS](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/70768574/78782214-951d-4ec7-afc9-d509fe68c82e)
![20230805_202151651_iOS](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/70768574/dac4d236-130c-4646-9513-37dc59e436d2)
![20230805_202205514_iOS](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/70768574/e9816e52-e0a0-4976-a0fc-d32468f5de38)

I looked at the hardware test. It's only got pictures on the webpage from a V1 PCb so I used the spare 5V and GND from H1A to test the board. I only had a 100mA 5V supply and this was dragged down to 2.75V and didn't power the ESP board. Will find a power supply with a higher capacity.

I will also try powering the ESP from a USB as per other posts.

In the mean time, any suggestions as to what I am doing wrong?

I have 2021 display and pump selected as the hardware. THe guide says Miami2021 but no reference to that on the web interface.

**Comment by visualapproach** [comment]
Hi, the Miami 2021 is legacy from earlier versions. Just a label but I should update the manual.
Check that the cables are connected to the right thing. Dsp to display and cio to the pump. And do the hardware test to get an idea if it's a connection issue. It's the same on all PCB versions. Connect the cables to each other and supply power.

**Comment by MattSpeke** [comment]
Hi,

Checked the cables. Have carried out resistance checks on all lines.
560 ohms beteween male and female JST cables through the resistors.
600 ohms through the level shifters from H1A and H2A to D1-7
0 ohms from 5v and GND on HA1 and HA2 to ESP header.

Hardware test fails with all failing.

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/70768574/9da950bb-ef67-40c9-aaeb-344317bc240c)

I have 2 other ESP boards so will program another boad and try it.

On the hardware Config menu there is a section for pinout. I have tried to change that as per the PCB with D1, D2 and D5 to CIO and D6, D4 and D7 to DSP. But every time I change it and restart the ESP it is back to the default pin outs. IS this a bug?

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/70768574/dcd3856a-345e-4ffe-93b1-b9af309f929c)

**Reply by visualapproach** [reply]
You have a v2b so set that option instead

**Reply by visualapproach** [reply]
And click save. It should remember it after reset

**Reply by MattSpeke** [reply]
Have set to V2B but still fails the hardware test.

**Reply by visualapproach** [reply]
Powered through 5V or USB? Not sure USB will work

**Reply by MattSpeke** [reply]
I have a 5V 2A supply to H1A pins on the board that I have the jumper wires on the "B" position

**Reply by MattSpeke** [reply]
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/70768574/fffbfcad-3685-4585-94fc-72de9ace59a1)

**Reply by visualapproach** [reply]
Ok and it says V2B after you saved now?

**Reply by MattSpeke** [reply]
Yes. Keeps the V2B option.

**Reply by visualapproach** [reply]
Cut out the 10k resistor. It's only for the other type of LLC. Don't know what it does to the red LLC but it won't help. Other than that, don't know. Did you perhaps mount one of the cables in reverse? Hard to tell with the all black theme

**Reply by MattSpeke** [reply]
Have cut out the resistor. still fails the test. Will replace the LLC as have 5 of everything!

The cables are connected:

5V - 5V
Gnd - Gnd
D1 - D6
D2 - D4
D5 - D3
NC - D7

**Reply by visualapproach** [reply]
Do that. And the other PCB you showed

**Reply by MattSpeke** [reply]
Ok. The second ESP works! Only thing I can think is that I looked at the firmware update page on the original board and I think there was a newer version of master and beta that I uploaded. The new board I have kept as the original firmware. I will reflash the original baord and try that.

Thanks very much for your help and prompt replies. Will shout you a coffee.

**Reply by visualapproach** [reply]
Any time! Glad it worked out. Thank you for the ☕ 👍
